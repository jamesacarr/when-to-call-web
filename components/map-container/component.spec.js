import React from 'react';
import { shallow } from 'enzyme';

import MapContainer from './component';

describe('<MapContainer/>', () => {
  it('renders correctly', () => {
    const classes = {
      container: 'container',
      map: 'map'
    };
    const result = { test: 'stuff' };
    const onLoad = () => {};

    const wrapper = shallow(<MapContainer classes={classes} result={result} onLoad={onLoad}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
