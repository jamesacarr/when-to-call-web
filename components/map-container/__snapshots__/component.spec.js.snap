// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MapContainer/> renders correctly 1`] = `
<withGoogleMap(Map)
  containerElement={
    <WithStyles(Paper)
      className="container"
    />
  }
  mapElement={
    <div
      className="map"
    />
  }
  onLoad={[Function]}
  result={
    Object {
      "test": "stuff",
    }
  }
/>
`;
